Core/resources-test/configTemplateTestData.csv

```diff
@@ -0,0 +1,6 @@
+site,building,phase,dataHall,equipmentType,device,equipmentNumber,template,host,slaveId
```

Comment:
These files in resources-test must be in a package corresponding to your test

Reply:
files moved to new package